src/OnboardingSPA/store/selectors.js

```diff
@@ -340,3 +340,23 @@ export function getTechSupportUrl( state ) {
 export function getPluginInstallHash( state ) {
 	return state.runtime.pluginInstallHash;
 }
+
+/**
+ * Gets the Queue Element on top
+ *
+ * @param {*} state
+ * @return {string} getQueuePeek
+ */
+export function getQueuePeek( state ) {
```

Comment:
do we need this as I don't see it being used anywhere else?

Reply:
I just added this for future use cases and this might come in handy later.